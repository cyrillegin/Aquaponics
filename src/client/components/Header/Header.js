import React from 'react';
import styled from '@emotion/styled';

const Header = () => {
  console.info('header');
  return <div>header</div>;
};

const styledHeader = styled(Header)``;

export default styledHeader;
