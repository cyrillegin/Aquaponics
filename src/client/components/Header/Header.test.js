import { mount } from 'enzyme';
import React from 'react';
import Header from './Header';

describe('Header', () => {
  it('should render a snap shot', () => {
    const wrapper = mount(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
