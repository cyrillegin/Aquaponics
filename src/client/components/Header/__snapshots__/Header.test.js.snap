// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render a snap shot 1`] = `ReactWrapper {}`;
